config.json

```
@@ -142,5 +142,9 @@
     },
     "kmip": {
         "providerName": "thales"
+    },
+    "apisLengthLimits": {
```

```suggestion
    "apisPostSizeLimits": {
```

I would suggest also to use api instead of apis as Limits is already plural ?

---

Also why we define them here if we have already default value (that are exactly the same) ?

---

I'm gonna update the name. This limit isnt even used for POST only, but also put. Actually this config.json here is more of an example config. In prod, users have a mechanism to update it. Here it's used as an example. When I did my test I used very low number to make sure it was working, and I kept it in the commit to demonstrate how it could be used